sklearn/tests/test_cross_validation.py

```diff
@@ -25,6 +25,8 @@
 with warnings.catch_warnings():
     warnings.simplefilter('ignore')
     from sklearn import cross_validation as cval
+    # We should be able to import this from cross_validation
+    from sklearn.cross_validation import FitFailedWarning as _FFW
```

Comment:
This is to test if `FitFailedWarning` can be imported from our old path (which was public and needs to be supported for atleast 2 more versions)

`from sklearn.cross_validation import FitFailedWarning as _` would be better in that case then.